src/components/SwitzerlandEstimatedCasesDivisionModal.tsx

```diff
@@ -87,26 +88,23 @@ export const SwitzerlandEstimatedCasesDivisionModal = ({
   show,
   handleClose,
 }: Props) => {
-  const [caseCounts, setCaseCounts] = useState<CaseCountEntry[] | undefined>();
+  //const [caseCounts, setCaseCounts] = useState<CaseCountEntry[] | undefined>();
   const [showSwissRegions, setShowSwissRegions] = useState(true);
-
   const { dateFrom, dateTo, country } = variantSampleSet.sampleSelector;
-  useEffect(() => {
-    let isSubscribed = true;
```

Comment:
I am not sure why we initialize `isSubscribed `to true here and it is checked in before `setCaseCounts` at the deleted line 100, which will always be true.